CHANGELOG.md

```diff
@@ -26,8 +26,9 @@ Mapbox welcomes participation and contributions from everyone.
 * Fix volatile tiles disappearing on "not modified" response ([#628](https://github.com/mapbox/mapbox-maps-ios/pull/628))
 * Fix crash in MapboxMap.clearData() ([#628](https://github.com/mapbox/mapbox-maps-ios/pull/628))
 * Trigger map redraw when feature state changes ([#628](https://github.com/mapbox/mapbox-maps-ios/pull/628))
-* Do not start background task if telemetry collection is disabled ([#](https://github.com/mapbox/mapbox-maps-ios/pull/))
-* Fix KVC decoding for iOS 15 ([#](https://github.com/mapbox/mapbox-maps-ios/pull/))
+* Do not start background task if telemetry collection is disabled ([#631](https://github.com/mapbox/mapbox-maps-ios/pull/631))
+* Fix KVC decoding for iOS 15 ([#631](https://github.com/mapbox/mapbox-maps-ios/pull/631))
```

Comment:
Thank you. I think there's another one somewhere

Reply:
Got that one too.